gramps/gui/views/treemodels/eventmodel.py

```diff
@@ -145,7 +145,7 @@ def column_place(self, data):
             return ""
 
     def column_type(self, data):
-        return str(EventType(data["type"]))
+        return EventType.get_str(data.type)
```

Comment:
Which style of attribute access is going to recommend going forwards?
DataDict as in this line `data.type` or standard dict[key] style as found a couple of lines below `data["gramps_id"]`

Reply:
In the PR to revise the filter system, I changed all to `data.attribute` which works for whatever you pass in (obj or data).

Reply:
Yes.  I agree that the `data.attribute` style is preferable.